### ISSUE TITLE
Verify true vs. file

### ISSUE THREAD

**Issue by adschm** [ChristianDresel/gatewaykxv2#10]
Wieso brauchen wir verify.sh, wenn

on verify true

da steht? Das file kann weg, oder?

**Comment by ChristianDresel** [comment]
die File ist für Blacklist von fastd-Clients gedacht:
https://forum.freifunk.net/t/fastd-peer-ip-blacklisting-am-server-fastd-conf/13212/11
ich würde sie gerne behalten falls man mal nen Peer blacklisten muss hat man gleich alles zur Verfügung und muss nicht erst umbauen

**Comment by adrianschmutzler** [comment]
Aber wäre es dann nicht besser, dann wie in der Wiki-Anleitung gleich
on verify "/etc/fastd/xxx/verify.sh";
zu verwenden? Die Datei gibt ja eh return 0 zurück, und wenn man es mal braucht, dann muss man nicht an zwei Stellen ändern (was man u.U. vergisst)?

**Comment by ChristianDresel** [comment]
stimmt, sollte man tun. Ich hatte da allerdings noch Probleme das sich peers nicht verbunden haben und hatte nie den Nerv den Fehler zu suchen und habs dann so gemacht, das es ging. Müsste man sich nochmal in Ruhe angucken
